index.bs

```diff
@@ -2958,6 +2960,8 @@ and a [=header list=] |newHeaders| (defaults to an empty [=list=]):
     ::  "`no-store`"
 1. Return |request|.
 
+Issue: Should |request|'s [=request/priority=] be "`low`"?
```

Comment:
@yoavweiss Could you comment on whether you think that **report** requests should also use low priority?